### ISSUE TITLE
Cannot install on Debian Buster

### ISSUE THREAD

**Issue by alexacallmebaka** [zerotier/ZeroTierOne#1223]
**Describe the bug**
I'm having some trouble installing Zerotier-cli on a Debian 10 laptop. When running the ```curl -s https://install.zerotier.com | sudo bash``` command from the website, I am met with the following output: 

```
*** ZeroTier One Quick Install for Unix-like Systems

* Tested distributions and architectures:
*   MacOS (10.7+) on x86_64 (just installs ZeroTier One.pkg)
*   Debian (7+) on x86_64, x86, arm, and arm64
*   RedHat/CentOS (6+) on x86_64 and x86
*   Fedora (16+) on x86_64 and x86
*   SuSE (12+) on x86_64 and x86
***   Mint (18+) on x86_64, x86, arm, and arm64

*** Please report problems to contact@zerotier.com and we will try to fix.

*** Detecting Linux Distribution

*** Found Debian "buster", or "sid" (or similar), creating /etc/apt/sources.list.d/zerotier.list
gpg: invalid key resource URL '/tmp/apt-key-gpghome.W8rxrr3ZT7/home:antergos.asc.gpg'
gpg: keyblock resource '(null)': General error
gpg: key DC30D7C23CBBABEE: 4 signatures not checked due to missing keys
gpg: key 4DFAB270CAA96DFA: 4 signatures not checked due to missing keys
gpg: key DCC9EFBF77E11517: 2 signatures not checked due to missing keys
gpg: key 7638D0442B90D010: 4 signatures not checked due to missing keys
gpg: key 9D6D8F6BC857C906: 4 signatures not checked due to missing keys
gpg: key CBF8D6FD518E17E1: 2 signatures not checked due to missing keys
gpg: key E0B11894F66AEC98: 5 signatures not checked due to missing keys
gpg: key EDA0D2388AE22BA9: 5 signatures not checked due to missing keys
gpg: key EF0F382A1A7B6500: 2 signatures not checked due to missing keys
gpg: key 7B1869016E94B574: 1 signature not checked due to a missing key
gpg: Total number processed: 13
gpg:       skipped new keys: 13

*** Installing zerotier-one package...
Hit:1 http://security.debian.org/debian-security buster/updates InRelease
Hit:2 http://deb.debian.org/debian buster InRelease
Hit:3 http://deb.debian.org/debian buster-updates InRelease
Get:4 http://download.zerotier.com/debian/buster buster InRelease [25.9 kB]
Hit:5 https://repo.nordvpn.com/deb/nordvpn/debian stable InRelease
Hit:6 http://packages.microsoft.com/repos/vscode stable InRelease
Err:4 http://download.zerotier.com/debian/buster buster InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
Reading package lists... Done
W: GPG error: http://download.zerotier.com/debian/buster buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
E: The repository 'http://download.zerotier.com/debian/buster buster InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  zerotier-one
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 707 kB of archives.
After this operation, 2,098 kB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  zerotier-one
E: There were unauthenticated packages and -y was used without --allow-unauthenticated

* Package installation failed! Unfortunately there may not be a package
* for your architecture or distribution. For the source go to:
*** https://github.com/zerotier/ZeroTierOne
```

**Expected behavior**
Zerotier to be installed on my system.

**Desktop (please complete the following information):**
 - OS: Linux
 - OS/Distribution Version: Debian Buster
 - ZeroTier Version: N/A
 - Hardware: Dell XPS 15

**Comment by alexacallmebaka** [comment]
I honestly think there is a problem with the Debian repo.

**Comment by laduke** [comment]
Hey we're looking at updating the script. But if you want to just get up and running, googling like "The following signatures couldn't be verified because the public key is not available" will tell you what to do. 
[for example](https://chrisjean.com/fix-apt-get-update-the-following-signatures-couldnt-be-verified-because-the-public-key-is-not-available/)

**Comment by mb-Ma** [comment]
I also meet this error. Did you fix it?

*** Found Ubuntu, creating /etc/apt/sources.list.d/zerotier.list
Hit:1 http://mirrors.tencentyun.com/ubuntu bionic InRelease
Hit:2 http://mirrors.tencentyun.com/ubuntu bionic-security InRelease                    
Hit:3 http://mirrors.tencentyun.com/ubuntu bionic-updates InRelease                      
Get:4 http://download.zerotier.com/debian/bionic bionic InRelease [20.5 kB]              
Err:4 http://download.zerotier.com/debian/bionic bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
Reading package lists... Done
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic-security/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic-updates/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://download.zerotier.com/debian/bionic/dists/bionic/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: GPG error: http://download.zerotier.com/debian/bionic bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
E: The repository 'http://download.zerotier.com/debian/bionic bionic InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
gpg is already the newest version (2.2.4-1ubuntu1.6).
0 upgraded, 0 newly installed, 0 to remove and 97 not upgraded.
gpg: WARNING: unsafe ownership on homedir '/home/ubuntu/.gnupg'

*** Installing zerotier-one package...
Hit:1 http://mirrors.tencentyun.com/ubuntu bionic InRelease
Hit:2 http://mirrors.tencentyun.com/ubuntu bionic-security InRelease                                     
Hit:3 http://mirrors.tencentyun.com/ubuntu bionic-updates InRelease                                      
Get:4 http://download.zerotier.com/debian/bionic bionic InRelease [20.5 kB]
Err:4 http://download.zerotier.com/debian/bionic bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
Reading package lists... Done
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic-security/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://mirrors.tencentyun.com/ubuntu/dists/bionic-updates/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://download.zerotier.com/debian/bionic/dists/bionic/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/zerotier-debian-package-key.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: GPG error: http://download.zerotier.com/debian/bionic bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1657198823E52A61
E: The repository 'http://download.zerotier.com/debian/bionic bionic InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package zerotier-one

*** Package installation failed! Unfortunately there may not be a package
*** for your architecture or distribution. For the source go to:
*** https://github.com/zerotier/ZeroTierOne
